modules/apps/layout/layout-content-page-editor-web/package.json

```diff
@@ -12,6 +12,7 @@
 		"@clayui/loading-indicator": "3.1.0",
 		"@clayui/sticker": "3.2.0",
 		"@clayui/tooltip": "3.3.5",
+		"@testing-library/react-hooks": "3.4.1",
```

Comment:
We must not add dev/test deps to project-local package.json files. And if we wanted to, they would belong in `devDependencies`, not `dependencies`; this in turn requires a lint suppression, which is a clue that you are doing it wrong.

I'm preparing https://github.com/liferay/liferay-npm-tools/issues/496 today to do this "the right way", so we'll be able to remove this line, and also line 26 below (which is another test dep).

I'm going to update the blacklist to catch at least this concrete instance of this problem in the future: https://github.com/liferay/liferay-npm-tools/blob/c680c6e8fa5ea2fc6e16f9be7fbad1abf732e293/packages/liferay-npm-scripts/src/presets/standard/index.js#L45